CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ project(oaknut LANGUAGES CXX VERSION 1.1.1)
 set(MASTER_PROJECT OFF)
 if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
   set(MASTER_PROJECT ON)
+  option(OAKNUT_USE_BUNDLED_CATCH "Use the embedded Catch2 submodule" OFF)
```

Comment:
It would probably be preferrable to have this option be on line 45 after `if (MASTER_PROJECT)`